### PR TITLE
BUGS-6353 Fix incorrect 4010 310 grammar

### PR DESCRIPTION
**Pull request by midhunkrishna**
Move L1 into LX and C3 inside L1

### CODE REVIEW

**Comment by midhunkrishna** [comment]
reference -> [EDI310_20171121.pdf](https://github.com/cloudlogistics/baldr/files/1635796/EDI310_20171121.pdf)

screenshot_011618_091912_pm

File: lib/baldr/grammar/version4010/set310.rb [hunk 1]
```diff
@@ -35,13 +35,14 @@ def record_defs
...
+       {id: 'L1', min: 0, max: 20, level:[
+          {id: 'C3', min: 0, max: 1}
+        ]}
       ]},
       {id: 'L3', min: 1, max: 1},
-      {id: 'L1', min: 0, max: 20},
-      {id: 'C3', min: 0, max: 1},
-      {id: 'SE', min: 1, max: 1},
+      {id: 'SE', min: 1, max: 1}
     ]
```
Comment (npersad84) [inline lib/baldr/grammar/version4010/set310.rb:46]: @midhunkrishna The substantive changes are good to go. One gripe. Spacing seems off between the N7 Loop and the L0 and L1. The latter 2 seem to be a one space to far to the left.

cc: @nathanielread , @scottkf

File: lib/baldr/grammar/version4010/set310.rb [hunk 2]
```diff
@@ -35,13 +35,14 @@ def record_defs
           {id: 'M7', min: 0, max: 5}
         ]},
        {id: 'L0', min: 0, max: 120, level:[
-         {id: 'L5', min: 0, max: 9999},
-       ]},
+          {id: 'L5', min: 0, max: 9999},
```
Comment (npersad84) [inline lib/baldr/grammar/version4010/set310.rb]: @midhunkrishna Can you please remove this comma here, I think I put that on accident last PR.

**Review by npersad84 — DISMISSED** [review]

**Comment by npersad84** [comment]
@midhunkrishna Failing specs :(

https://circleci.com/gh/cloudlogistics/baldr/89?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Review by npersad84 — APPROVED** [review]
@midhunkrishna Commit message has a typo if you want to change it before we merge it.

**Pull request merged**